Gemfile

```diff
@@ -18,6 +18,8 @@ gem 'sprockets-rails'
 # Use postgresql as the database for Active Record
 gem 'pg', '~> 1.1'
 
+gem 'rails-controller-testing'
+
```

Comment:
- [ ] Please add RSpec gem for testing  be similar to the example below :arrow_down:  It needs to be in the :development group to expose generators and rake tasks without having to type RAILS_ENV=test. :+1: 

```
group :development, :test do
 ...
 gem 'rspec-rails'
end

```